app/services/container_dashboard_service.rb

```diff
@@ -228,6 +229,27 @@ def daily_network_metrics
     end
   end
 
+  def hourly_pods_metrics
+    resource_ids = @ems.present? ? [@ems.id] : ManageIQ::Providers::ContainerManager.select(:id)
+    hourly_pods_trend = Hash.new(0)
+    hourly_metrics =
+        MetricRollup.find_all_by_interval_and_time_range("hourly", 1.day.ago.beginning_of_hour.utc, Time.now.utc)
+    hourly_metrics =
+        hourly_metrics.where('resource_type = ? AND resource_id in (?)', 'ExtManagementSystem', resource_ids)
+
+    hourly_metrics.each do |m|
+      hour = m.timestamp.beginning_of_hour.utc
+      hourly_pods_trend[hour] += m.stat_containergroup_create_rate
+    end
+
+    if hourly_metrics.any?
+      {
+          :xData => hourly_pods_trend.keys,
+          :yData => hourly_pods_trend.values.map(&:round)
+      }
+    end
+  end
+
```

Comment:
@kbrock Assuming nothing has changed on the metrics infrastructure, this part should be ok.